examples/initialize_from_array/run_from_array.py

```diff
@@ -0,0 +1,108 @@
+import numpy as np
+
+try:
+    import torch
+
+    on_gpu = torch.cuda.is_available()
```

Comment:
Using `torch` for this is a wildly huge dependency :) I think you meant to use
```py
from impactx import Config, ImpactX, elements
```
and then use `Config.have_gpu`.